oracledb-ballerina/gradlew

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
```

Comment:
we only need `gradlew` file at root level

Reply:
Fixed with ae9c94747102126fe586ad09fa1f73d0652e8fcd